tests/backends/base/test_creation.py

```diff
@@ -67,3 +70,45 @@ def test_migrate_test_setting_true(self, mocked_migrate, mocked_ensure_connectio
             mocked_migrate.assert_called_once()
         finally:
             connection.settings_dict = saved_settings
+
+
+class TestTestDbSerialization(SimpleTestCase):
+    databases = ['default']
+
+    def test_serialize_circular_ref(self):
+        import django
+        print(django.conf.settings.INSTALLED_APPS)
```

Comment:
Please remove debug prints.

Reply:
W00ps, seems I missed one. Removed it :-)